tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -610,3 +625,299 @@ def test_pdb_check_simultaneous_node_drains(
+                config.switch_to_provider()
+            except ClusterNotFoundException:
+                pytest.skip("Provider cluster not found in multicluster config")
+
+        if not config.DEPLOYMENT.get("ec_default_pools"):
+            pytest.skip("Test runs only on EC pools")
+
+        self.sanity_helpers = Sanity()
+        self.stopped_node_objs = []
+        self._nodes = nodes
+
+        def teardown():
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `teardown`

Add return type annotation: `None`

(ANN202)